frontend/src/app/note/note.component.spec.ts

```diff
@@ -133,6 +133,10 @@ describe('NoteComponent', () => {
       tester.detectChanges();
 
       expect(tester.form).toBeTruthy();
+      // this is probably a regression in Jasmine v3
+      // as the matchers should already be available but aren't
+      // runs fine without it in Jasmine 2.99 but not in 3.1
+      jasmine.addMatchers(speculoosMatchers);
```

Comment:
This is strange. As is, the test passes, but with a log saying that this test doesn't have any expectation. If I change one of the expectation to make it fail, it fails though. And my guess is that the proper fix is to use async (since we have expectations inside a promise callback, and they're thus checked asynchronously). But then, why does the test fail if we omit the async?